app/src/main/java/org/oppia/app/help/faq/faqsingle/FAQSingleActivity.kt

```diff
@@ -0,0 +1,33 @@
+package org.oppia.app.help.faq.faqsingle
+
+import android.content.Context
+import android.content.Intent
+import android.os.Bundle
+import org.oppia.app.activity.InjectableAppCompatActivity
+import javax.inject.Inject
+
+/** The FAQ page activity for placement of single FAQ. */
+class FAQSingleActivity : InjectableAppCompatActivity() {
```

Comment:
Please change toolbar title to FAQ instead of FAQs... as per the discussion we had.

Reply:
I have updated all file names but tool bar title is FAQs in both the activities which is fine I think. Anyways that is a part of Highfi too so that will get clarified with Chantel if needed.